cfrontend/C2C.ml

```diff
@@ -717,8 +717,7 @@ let rec convertExpr env e =
       let e2' = convertExpr env e2 in
       if Cutil.is_composite_type env e1.etyp
       && List.mem AVolatile (Cutil.attributes_of_type env e1.etyp) then
-        warning "assignment to a l-value of volatile composite type. \
-                 The 'volatile' qualifier is ignored.";
+        warning Cerrors.Unnamed "assignment to an lvalue of volatile composite type";
```

Comment:
The part "The volatile qualifier is ignored" is important and must stay.  Otherwise it's unclear why there is a warning in the first place.  (This is a CompCert limitation.)